test_utils/fake_catalog_api.py

```diff
@@ -910,6 +910,7 @@
     "short_description": None,
     "subject_uuids": [],
     "transcript_languages": [],
+    "course_runs": [],
```

Comment:
I think `discovery` will never miss this `course_runs` key.  that's why added all places where it was missing.